README.md

```diff
@@ -14,6 +14,16 @@ React component to load formsort flows.
 
 Embed flows within other webpages, allowing communication between the embed and the parent page.
 
+## Publishing
+
+This repo is published using [Craft](https://github.com/getsentry/craft).
+
+You need to set the following environment variables to be able to work with Craft:
+
+1. `GITHUB_TOKEN`: [Create a personal access token](https://github.com/settings/tokens/new?scopes=repo&description=Craft) from GitHub with `repo` permissions and set the token value to this env variable
+2. `NPM_TOKEN`: Create an automation token from NPM by visiting https://www.npmjs.com/settings/<npm_user>/tokens/new and select **Automation** as the token type.
+
+Then all you need to do is `yarn workspace @formsort/<package> run release <version>`. We'll take it from here :)
```

Comment:
should this be run from the root of the project?

Reply:
Yes (or you can run `yarn release <version>` from the package sub-directory. I'll add a note for this.

Reply:
```suggestion
Then all you need to do is `yarn workspace @formsort/<package> run release <version>` from the repo root or `yarn release <version>` from the package directory you want to release. We'll take it from here :)
```